### ISSUE TITLE
[META] Feature Proposal Form

### ISSUE THREAD

**Issue by djrenren** [rit-sse/api#40]
I think we should have a list of things required for a feature proposal.

My suggestion:
- What data are we storing
- How are we going to populate it
- Where will we actually use the data
- How will we query the data

Seem reasonable?

**Comment by bbesmanoff** [comment]
:+1: :+1:

Also have somebody that isn't the author sign off on it

**Comment by kristenmills** [comment]
:+1:

**Comment by kocsenc** [comment]
Where put these things?

**Comment by djrenren** [comment]
yup that's the idea. Just to make sure we have properly reasoned through and agreed upon features before we merge them in
